.github/workflows/codespaces.yml

```diff
@@ -0,0 +1,28 @@
+name: codspeed-benchmarks
```

Comment:
feels like this file should be named `.github/workflows/codspeed.yml`?

Reply:
True going to change it